packages/now-cli/test/integration.js

```diff
@@ -297,6 +299,30 @@ test('deploy using only now.json with `redirects` defined', async t => {
   t.is(location, 'https://example.com/foo/bar');
 });
 
+test('deploys gatsby and prints cached directories', async t => {
+  const directory = example('gatsby');
+  const packageJsonPath = path.join(directory, 'package.json');
+  const packageJsonOriginal = await readFile(packageJsonPath, 'utf8');
+  const pkg = JSON.parse(packageJsonOriginal);
+
+  async function tryDeploy(cwd) {
+    await execa(binaryPath, [...defaultArgs, '--public', '--confirm'], {
+      cwd,
+      stdio: 'inherit',
+      reject: true,
+    });
+
+    t.true(true);
+  }
+
+  await tryDeploy(directory);
+  await sleep(5000);
```

Comment:
Why 5 seconds?

Reply:
It seemed like the deployment was returning a 0 status code prior to it being completely done but perhaps that was a different issue. Let me try again without the delay.

Reply:
Ok so I updated to 60 seconds because S3 is terrible and we have to wait for the cache to propagate (yes the cache response is cached 🤦‍♂️ )